mkdocs/index.md

```diff
@@ -275,22 +270,22 @@ that expresses the data processing/cleaning routine,
 such that it accepts a dataframe as the first argument,
 and returns a modified dataframe:
 
-.. code-block:: python
-
-    import pandas_flavor as pf
+```python
+import pandas_flavor as pf
 
-    @pf.register_dataframe_method
-    def my_data_cleaning_function(df, arg1, arg2, ...):
-        # Put data processing function here.
-        return df
+@pf.register_dataframe_method
+def my_data_cleaning_function(df, arg1, arg2, ...):
+    # Put data processing function here.
+    return df
+```
 
 We use [`pandas_flavor`](https://github.com/Zsailer/pandas_flavor) to register the function natively on a `pandas.DataFrame`.
 
 
 ### Add a test case
 
 Secondly, we ask that you contribute a test case,
-to ensure that it works as intended.
+to ensure that the function works as intended.
 Follow the [contribution] docs for further details.
 
 [contribution]: https://pyjanitor-devs.github.io/pyjanitor/contributing.html#unit-test-guidelines
```

Comment:
Any idea which page this "Contribution" link should point to? @ericmjl

Reply:
Hmmm, I think we've dropped that doc. Let's delete, I guess.